cmd/internal/cli/actions_linux.go

```diff
@@ -352,9 +358,9 @@ func execStarter(cobraCmd *cobra.Command, image string, args []string, name stri
 			}
 			generator.SetProcessArgs([]string{"/sbin/init"})
 		}
-		pwd, err := user.GetPwUID(uint32(os.Getuid()))
+		pwd, err := user.GetPwUID(uint32(uid))
```

Comment:
note that on line 159 uid changed. Just making sure that that is not passing unnoticed.

Reply:
Good catch ! I will revert this